DataLayer/Migrations/Scripts/20190520094439_DescriptionTab_Up.sql

```diff
@@ -0,0 +1,36 @@
+﻿DROP TRIGGER IF EXISTS TR_GameEvent_InsteadOfDelete;
+
+GO
+
+CREATE TRIGGER TR_GameEvent_InsteadOfDelete
+  ON GameEvent
+  INSTEAD OF DELETE
+AS
+  BEGIN
+    DELETE FROM GameEventParticipation
+    FROM DELETED
+    WHERE GameEventParticipation.TakesPartInId = DELETED.Id;
+
+    DELETE FROM Game -- This DELETE could be replaced with CascadeDelete on Games, but it seems more consistent to put everything in one place.
+    FROM DELETED
+    WHERE Game.GameEventId = DELETED.Id;
+
+    DELETE FROM GameEvent
+    FROM DELETED
+    WHERE GameEvent.Id = DELETED.Id;
+
+	DELETE FROM DescriptionTab
+	FROM DELETED
+	WHERE DescriptionTab.GameEventId = DELETED.Id;
+
+  END;
+
+GO
+
+INSERT 
+INTO DescriptionTab(GameEventId)
+SELECT TMP.Id From 
+	(SELECT Id FROM GameEvent EXCEPT 
+	 SELECT GameEventId as ID FROM DescriptionTab) as TMP;
```

Comment:
In current model every GameEvent has a DescriptionTab. Therefore I need to create DescriptionTabs for older GameEvents. This Except statement may be redundant, because if we've just created a table DescriptionTab then it must be empty.

Reply:
I answered that question by myself, but `TMP` isn't a decriptive name at all, so change it to something meaningful.

Reply:
Done.